### PR TITLE
Setting up two Besu test networks

### PR DESCRIPTION
**Pull request by dhinakaran2705**

### CODE REVIEW

File: tests/network-setups/besu/scripts/setupNetwork.sh [hunk 1]
```diff
@@ -36,46 +38,57 @@ cd ../../../../
...
 # User instructions
 #echo "To complete the setup of the SampleBesuNetwork do the following:" 
 #echo "Open five terminals and run these:"
-echo "Five tmux sessions will be started to run EthSigner and the four validator nodes:"
+echo "Five tmux sessions will be started for Network $1 to run EthSigner and the four validator nodes: EthSigner_session and Node1_session to Node4_session"
 
-tmux new-session -d -s EthSigner_session 'cd  SampleBesuNetwork; ethsigner --chain-id=1337 --downstream-http-port=8590 multikey-signer --directory=keys/'
+#chainId=$(($1 + 1336)) # chainId starts from 1337 for Network 1 and increments by 1 for each network
+#downstreamPort=$((1000*$1 + 7590)) #downstream port i.e., port for Node-1, is 8590 for Network 1 and increments by 1000 for each network
+#tmux new-session -d -s EthSigner_session 'ethsigner --chain-id='${chainId}' --downstream-http-port='${downstreamPort}' multikey-signer --directory=keys/'
```
Comment (VRamakrishna) [inline tests/network-setups/besu/scripts/setupNetwork.sh:75]: Can you delete commented out lines like these if you don't need them for the future?

File: tests/network-setups/besu/Makefile [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+start-network1:
+	bash scripts/setupNetwork.sh 1
+
+start-network2:
+	bash scripts/setupNetwork.sh 2
 
 test:
-	bash scripts/testSetup.sh
+	bash scripts/testSetup.sh 1 8545
+	bash scripts/testSetup.sh 2 9544
```
Comment (VRamakrishna) [inline tests/network-setups/besu/Makefile:11]: Why not 9545 (8545 + 100)?

**Review by VRamakrishna — APPROVED** [review]

File: tests/network-setups/besu/Makefile (hunk 2 above)
Comment (dhinakaran2705) [reply tests/network-setups/besu/Makefile:11]: 9545 is taken by Truffle

File: tests/network-setups/besu/scripts/setupNetwork.sh (hunk 1 above)
Comment (dhinakaran2705) [reply tests/network-setups/besu/scripts/setupNetwork.sh:75]: Yeah, will do that. Thanks for pointing it out.

**Pull request merged**